pulp_ansible/app/tasks/collections.py

```diff
@@ -717,6 +717,9 @@ async def _post_save(self, batch):
             docs_blob = d_content.extra_data.get("docs_blob", {})
             if docs_blob:
                 collection_version.docs_blob = docs_blob
+            execution_environment = d_content.extra_data.get("execution_environment", {})
```

Comment:
`execution_environment` needs to be set on `d_content.extra_data`
https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/tasks/collections.py#L397-L405

Reply:
Thanks, thought I had removed this.  Seems ctrl-z wasn't my friend earlier today.